Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -1180,6 +1206,12 @@ eval (PopUpModalAction (PopUpModal.OnButton2Click)) state = case state.props.isP
   NoPopUp -> continue state
   ActiveQuotePopUp -> exit $ CheckCurrentStatus
 
+eval (PopUpModalAction (PopUpModal.Tipbtnclick idx value)) state = case state.props.isPopUp of
+    TipsPopUp -> continue state{props{customerTip{tipActiveIndex = idx, tipForDriver= value, isTipSelected = if idx == 0 then false else true}}}
```

Comment:
rename `idx` 
use only index == 0, it will give boolean only,  no need to specify explicitly